packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -185,7 +190,12 @@ const processChatMessageContent = (message: ChatMessageWithStatus): string | und
 const generateImageAttachmentImgHtml = (message: ChatMessageWithStatus, attachment: ChatAttachment): string => {
   if (attachment.previewUrl !== undefined) {
     const contentType = extractAttachmentContentTypeFromName(attachment.name);
-    const src = message.resourceCache?.[attachment.previewUrl].sourceUrl ?? '';
+    const resourceCache = message.resourceCache?.[attachment.previewUrl];
+    let src = URL.createObjectURL(new Blob());
```

Comment:
Why do we need to create a new empty blob here?
How do you free this up later?

Reply:
Why can this not just be an empty string?

Reply:
Resolved with using just a static string of `blob://` for now